benchbuild/source/git.py

```diff
@@ -0,0 +1,121 @@
+"""
+Declare a git source.
+"""
+import typing as tp
+
+import attr
+import plumbum as pb
```

Comment:
Codacy found an issue: [Unable to import 'plumbum'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)